vm/systemSmartContracts/staking_test.go

```diff
@@ -1981,3 +1983,11 @@ func checkIsStaked(t *testing.T, sc *stakingSC, callerAddr, stakerPubKey []byte,
 	retCode := sc.Execute(arguments)
 	assert.Equal(t, expectedCode, retCode)
 }
+
+func TestBech32FromByte(t *testing.T) {
```

Comment:
No assert here

Reply:
i would leave it here - as it is easy to create erd addresses for testing purposes.

Reply:
All tests should have at least one **relevant** assert, otherwise please move the code in the `example` package. If you move the code to the example package then you can print all system SCs addresses with nice description.

Reply:
Done